jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildTarTask.java

```diff
@@ -88,16 +90,26 @@ public FileCollection getInputFiles() {
    */
   @OutputFile
   public String getOutputFile() {
-    return getTargetPath();
+    return getTarPath().toString();
   }
 
   /**
-   * Returns the output directory for the tarball. By default, it is {@code build/jib-image.tar}.
+   * Returns the location for the tarball. By default, it is {@code build/jib-image.tar}.
+   *
+   * @return the tarball location
+   */
+  private Path getTarPath() {
+    return getProject().getBuildDir().toPath().resolve("jib-image.tar");
+  }
+
+  /**
+   * Returns the output directory for the image digest. By default, it is {@code
+   * build/jib-image.digest}.
    *
    * @return the output directory
    */
-  private String getTargetPath() {
```

Comment:
nit: looks like this was in the wrong location before - let's move these private methods to the bottom.